client/protocols/analytics.py

```diff
@@ -36,14 +36,19 @@ def run(self, messages):
         """Returns some analytics about this autograder run."""
         statistics = {}
         statistics['time'] = str(datetime.now())
+        statistics['time-utc'] = str(datetime.utcnow())
         statistics['unlock'] = self.args.unlock
 
         if self.args.question:
-            # TODO(denero) Get the canonical name of the question
-            statistics['question'] = self.args.question
+            statistics['question'] = [t.name for t in self.assignment.specified_tests]
+            statistics['question-args'] = self.args.question
```

Comment:
It's technically a list of multiple questions (to support multiple uses of `-q`) 

`requested-questions` sounds good.